nipype/interfaces/base.py

```diff
@@ -72,7 +72,7 @@ class Str(traitlets.Unicode):
 #traits.Str = Str
 
 
-# dj TOTHINK: do i need it?
+# dj TOASK: nipype almost doest use it, should we remove?
```

Comment:
sure, but someone has to use it ;-) I was simply marking things when I saw that they are (almost) not used.

However I agree, that customized errors are nice and they also should be tested better. I found a few places that the raised exception was probably not the one that was expected, but the test was passing.